packages/chart/src/XYAxis.ts

```diff
@@ -580,6 +608,8 @@ export class XYAxis extends SVGWidget {
     orientation: publish<this, string>;
     orientation_default: publish<this, string>;
 
+    @publish(false, "boolean", "2D Range Selector")
+    selectionMode2D: publish<this, boolean>;
     @publish(false, "boolean", "Range Selector")
     selectionMode: publish<this, boolean>;
```

Comment:
Should we have a single PP for both of these?  "selectionMode" with options "1D", "2D"?

Reply:
I thought about this but didn't want to make this a potentially breaking change.